modules/enabling-anonymous-pulls-upgraded-cluster.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * registry/accessing-the-registry.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="enabling-anonymous-pulls-upgraded-cluster_{context}"]
+= Anonymous pulls for upgraded clusters
+
+Use the following procedure to enable anonymous pulls if you have upgraded to {product-title} 4.16 or later from {product-title} 4.15 or earlier.
+
+.Procedure 
+
+. Enter the following command to see if the `system:unauthenticated` group is included in the `subjects` parameter of your `ClusterRoleBinding` object:
```

Comment:
The idea with adding this verification was to not have to distinguish between new and existing clusters - if the user always checks if `self-access-reviewers` have are available to `system:unauthenticated` then this alone can decide whether they need execute procedure "Create a file named `add-self-access-review-unauth.yaml` and add the following content" or not.
WDYT?